Sift/Schema/create_order.json

```diff
@@ -11,6 +11,12 @@
     "$user_id": {
       "type": [ "string", "null" ]
     },
+    "$time": {
```

Comment:
`$time` and `$ip` are optional fields and I don't believe they should be part of the json schema. All events already extend `SiftEvent`, which has a function called `AddTime`. If you'd like, you can open a PR to add a new function to `SiftEvent` called `AddIp`, but I think it should be separate from this PR.

Reply:
sure. I am going to remove them